src/providers/address-book/address-book.ts

```diff
@@ -83,8 +83,10 @@ export class AddressBookProvider {
             if (ab && _.isString(ab)) ab = JSON.parse(ab);
             ab = ab || {};
             _.each(ab, contact => {
-              const ctc = this.getContact(contact);
-              if (ctc) contacts.push(ctc);
+              if (contact.address) {
```

Comment:
What happen if a contact doesn't have address? I think it's mandatory when creating new contact.

Reply:
+1 (it's only for migration to new address book)